File: Auriculoterapia.Api/Domain/Usuario.cs
namespace Auriculoterapia.Api.Domain
{
    public class Usuario 
    {
        public int Id {get; set;}
        public string Nombre {get; set;}
        public string Apellido {get; set;}
        public string Email {get; set;}
        public string Contrasena {get; set;}
        public string NombreUsuario {get; set;}
        public string Sexo {get; set;}
        public string PalabraClave {get; set;}
        public string Token {get; set;}

        //OneToOne Relationship
        //public int PacienteId {get; set;}
        public Paciente Paciente {get; set;}

        //public int EspecialistaId {get; set;}
        public Especialista Especialista {get; set;}
        
    }
}

File: Auriculoterapia/app/src/main/java/com/example/android/auriculoterapia_app/models/Usuario.kt
package com.example.android.auriculoterapia_app.models

import java.io.Serializable

data class Usuario
    (
    var id: Int,
    var nombre: String,
    var apellido: String,
    var email: String,
    var contrasena: String,
    var nombreUsuario: String,
    var sexo: String,
    var palabraClave: String,
    var token : String
): Serializable{

}